rct229/rules/section22/section22rule11.py

```diff
@@ -0,0 +1,120 @@
+            each_rule=Section22Rule11.ChillerFluidLoopRule(),
+            index_rmr="baseline",
+            id="22-11",
+            description="For Baseline chilled-water system that does not use purchased chilled water, variable-flow secondary pump shall be modeled as 13W/gpm at design conditions.",
+            list_path="ruleset_model_instances[0].fluid_loops[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmr_baseline = context.baseline
+        rmi_b = rmr_baseline["ruleset_model_instances"][0]
+
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+        # create a list contains all HVAC systems that are modeled in the rmi_b
+        available_type_list = [
+            hvac_type
+            for hvac_type in baseline_system_types_dict.keys()
+            if len(baseline_system_types_dict[hvac_type]) > 0
+        ]
+
+        primary_secondary_loop_dict = get_primary_secondary_loops_dict(rmi_b)
+
+        return (
+            any(
+                [
+                    available_type in APPLICABLE_SYS_TYPES
+                    for available_type in available_type_list
+                ]
+            )
+            and primary_secondary_loop_dict
+        )
+
+    def create_data(self, context, data):
+        rmr_baseline = context.baseline
+        rmi_b = rmr_baseline["ruleset_model_instances"][0]
+
+        primary_secondary_loop_dict = get_primary_secondary_loops_dict(rmi_b)
+
+        return {"primary_secondary_loop_dict": primary_secondary_loop_dict}
+
+    def list_filter(self, context_item, data):
+        fluid_loop_b = context_item.baseline
+        primary_secondary_loop_dict = data["primary_secondary_loop_dict"]
+
+        return fluid_loop_b["id"] in primary_secondary_loop_dict.keys()
+
+    class ChillerFluidLoopRule(RuleDefinitionListIndexedBase):
+        def __init__(self):
+            super(Section22Rule11.ChillerFluidLoopRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                each_rule=Section22Rule11.ChillerFluidLoopRule.SecondaryChildLoopRule(),
+                index_rmr="baseline",
+                list_path="$..child_loops[*]",
+            )
+
+        class SecondaryChildLoopRule(RuleDefinitionBase):
+            def __init__(self):
+                super(
+                    Section22Rule11.ChillerFluidLoopRule.SecondaryChildLoopRule, self
+                ).__init__(
+                    rmrs_used=UserBaselineProposedVals(False, True, False),
+                    required_fields={
+                        "$": ["pump_power_per_flow_rate"],
+                    },
+                )
+
+            def get_calc_vals(self, context, data=None):
+                child_loop_b = context.baseline
+                secondary_loop_pump_power_per_flow_rate = child_loop_b[
+                    "pump_power_per_flow_rate"
+                ]
+                req_pump_flow_rate = 13 * ureg("W/gpm")
```

Comment:
It would be better to make this a constant near the top of the file.

Reply:
Fixed.